nodeSidebar.json

```diff
@@ -64,6 +64,7 @@
 	"ETL > Transform > DSP": [
 		"nodes/TRANSFORMERS/SIGNAL_PROCESSING/BUTTER/BUTTER",
 		"nodes/TRANSFORMERS/SIGNAL_PROCESSING/FIR/FIR",
+		"nodes/TRANSFORMERS/SIGNAL_PROCESSING/FFT/FFT",
```

Comment:
Instead of manually adding node's md file path here we should add it's type to `nodes_sidebar_map.json` file and then autogenerate this file using this command: `python3 generate_nodesidebar.py`